[32m2024-02-01 17:10:44 +0100[0m - dagster - [34mDEBUG[0m - __ASSET_JOB - 09de610b-f598-4c5b-b667-00d99bbf2f04 - 33951 - LOGS_CAPTURED - Started capturing logs in process (pid: 33951).
[32m2024-02-01 17:10:44 +0100[0m - dagster - [34mDEBUG[0m - __ASSET_JOB - 09de610b-f598-4c5b-b667-00d99bbf2f04 - 33951 - combined_and_cleaned_video_docs - STEP_START - Started execution of step "combined_and_cleaned_video_docs".
[32m2024-02-01 17:10:44 +0100[0m - dagster - [34mDEBUG[0m - __ASSET_JOB - 09de610b-f598-4c5b-b667-00d99bbf2f04 - combined_and_cleaned_video_docs - Loading file from: /Users/szymonzmyslony/projects/master-pipeline/pipeline/.dagster/storage/complete_video_content using PickledObjectFilesystemIOManager...
[32m2024-02-01 17:10:44 +0100[0m - dagster - [34mDEBUG[0m - __ASSET_JOB - 09de610b-f598-4c5b-b667-00d99bbf2f04 - 33951 - combined_and_cleaned_video_docs - ASSET_OBSERVATION - DagsterEventType.ASSET_OBSERVATION for step combined_and_cleaned_video_docs
[32m2024-02-01 17:10:44 +0100[0m - dagster - [34mDEBUG[0m - __ASSET_JOB - 09de610b-f598-4c5b-b667-00d99bbf2f04 - 33951 - combined_and_cleaned_video_docs - LOADED_INPUT - Loaded input "complete_video_content" using input manager "io_manager"
[32m2024-02-01 17:10:44 +0100[0m - dagster - [34mDEBUG[0m - __ASSET_JOB - 09de610b-f598-4c5b-b667-00d99bbf2f04 - 33951 - combined_and_cleaned_video_docs - STEP_INPUT - Got input "complete_video_content" of type "Any". (Type check passed).
[32m2024-02-01 17:10:44 +0100[0m - dagster - [34mINFO[0m - __ASSET_JOB - 09de610b-f598-4c5b-b667-00d99bbf2f04 - combined_and_cleaned_video_docs - https://www.ycombinator.com/library/GV-should-you-start-a-startup
[32m2024-02-01 17:10:44 +0100[0m - dagster - [34mERROR[0m - [31m__ASSET_JOB - 09de610b-f598-4c5b-b667-00d99bbf2f04 - 33951 - combined_and_cleaned_video_docs - STEP_FAILURE - Execution of step "combined_and_cleaned_video_docs" failed.

dagster._core.errors.DagsterExecutionStepExecutionError: Error occurred while executing op "combined_and_cleaned_video_docs"::

TypeError: chunk_document_into_clean_paragraphs() missing 1 required positional argument: 'logger'

Stack Trace:
  File "/Users/szymonzmyslony/Library/Caches/pypoetry/virtualenvs/pipeline-RtkjAAJa-py3.11/lib/python3.11/site-packages/dagster/_core/execution/plan/utils.py", line 54, in op_execution_error_boundary
    yield
  File "/Users/szymonzmyslony/Library/Caches/pypoetry/virtualenvs/pipeline-RtkjAAJa-py3.11/lib/python3.11/site-packages/dagster/_utils/__init__.py", line 467, in iterate_with_context
    next_output = next(iterator)
                  ^^^^^^^^^^^^^^
  File "/Users/szymonzmyslony/Library/Caches/pypoetry/virtualenvs/pipeline-RtkjAAJa-py3.11/lib/python3.11/site-packages/dagster/_core/execution/plan/compute_generator.py", line 131, in _coerce_op_compute_fn_to_iterator
    result = invoke_compute_fn(
             ^^^^^^^^^^^^^^^^^^
  File "/Users/szymonzmyslony/Library/Caches/pypoetry/virtualenvs/pipeline-RtkjAAJa-py3.11/lib/python3.11/site-packages/dagster/_core/execution/plan/compute_generator.py", line 125, in invoke_compute_fn
    return fn(context, **args_to_pass) if context_arg_provided else fn(**args_to_pass)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/szymonzmyslony/projects/master-pipeline/pipeline/pipeline/assets.py", line 275, in combined_and_cleaned_video_docs
    cleaned_documents = [
                        ^
  File "/Users/szymonzmyslony/projects/master-pipeline/pipeline/pipeline/assets.py", line 280, in <listcomp>
    chunk_document(video)
  File "/Users/szymonzmyslony/projects/master-pipeline/pipeline/pipeline/assets.py", line 266, in chunk_document
    transcript_chunker = DocumentChunker(
                         ^^^^^^^^^^^^^^^^
  File "/Users/szymonzmyslony/projects/master-pipeline/pipeline/chunker/DocumentChunker.py", line 10, in __init__
    paragraphs = chunk_document_into_clean_paragraphs(document)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[0m
